_episodes/17-responsiblity.md

```diff
@@ -1,74 +1,243 @@
 
-* Most clusters have a queue available for interactive use called 
-  {% for queue in site.workshop_interactive_queues %}{{ queue }}, {% endfor %} etc.  
-  This is generally a better way to run small tests or compile programs.
+Don’t run jobs on the login node (though quick tests are generally fine). A “quick test” is
+generally anything that uses less than 4GB of memory, 4 CPUs, and 10 minutes of time. If you
+use too much resource then other users on the login node will start to be affected - their
+login sessions will start to run slowly and may even freeze or hang. 
+
+> ## Login nodes are a shared resource
+>
+> Remember, the login node is shared with all other users and your actions could cause
+> issues for other people. Think carefully about the potential implications of issuing
+> commands that may use large amounts of resource.
+>
+{: .callout}
+
+Remember, you can always use the commands `top` and `ps ux` to list the processes you are running on a login
+node and the amount of CPU and memory they are using. The `kill` command can be used along
+with the *PID* to terminate any processes that are using large amounts of resource.
 
 > ## Login Node Etiquette
 > 
 > Which of these commands would probably be okay to run on the login node?
-> 1. python physics_sim.py
-> 2. make
-> 3. create_directories.sh
-> 4. molecular_dynamics_2
-> 5. tar -xzf R-3.3.0.tar.gz
+> python physics_sim.py
+> make
+> create_directories.sh
+> molecular_dynamics_2
+> tar -xzf R-3.3.0.tar.gz
+> 
 {: .challenge}
 
-* If someone is being inappropriate and using the login node to run all of their stuff, message an
-  administrator to take a look at things and deal with them.
+If you experience performance issues with a login node you should report it to the system
+staff (usually via the helpdesk) for them to investigate. You can use the `top` command
+to see which users are using which resources.
 
 ## Test before scaling
 
-* Before submitting a large run of jobs, submit one as a test first to make sure everything works.
+Remember that you are generally charged for usage on shared systems. A simple mistake in a 
+job script can end up costing a large amount of resource budget. Imagine a job script with 
+a mistake that makes it sit doing nothing for 24 hours on 1000 cores or one where you have
+requested 2000 cores by mistake and only use 100 of them! This problem can be compounded 
+when people write scripts that automate job submission (for example, when running the same
+calculation or analysis over lots of different input).  When this happens it hurts both you
+(as you waste lots of charged resource) and other users (who are blocked from accessing the
+idle compute nodes).
+
+On very busy resources you may wait many days in a queue for your job to fail within 10 seconds
+of starting due to a trivial typo in the job script. This is extremely frustrating! Most
+systems provide small, short queues for testing that have short wait times to help you 
```

Comment:
the concept of queues may not exist on the cluster at hand. Maybe just say `Most systems provide smaller resources for testing`

Reply:
Addressed https://github.com/hpc-carpentry/hpc-intro/pull/101/commits/152f118c823ddf914b2c9b09035af1a649f8cc4d